lib/ModelMixins/GeojsonMixin.ts

```diff
@@ -602,6 +604,33 @@ function GeoJsonMixin<T extends AbstractConstructor<BaseType>>(Base: T) {
           });
         } else {
           const dataSource = await this.loadGeoJsonDataSource(geoJsonWgs84);
+
+          if (this.clustering.enabled) {
+            const pinBackgroundColor = this.clustering.pinBackgroundColor;
```

Comment:
I don't know anything about `runInAction(...)`, but a lot of the surrounding code uses that for accessing things in `this`. Does this code not need that for some reason?

Reply:
Hi @pjonsson,
`runInAction`, like `action`,  is required when you modify an `observable`. 
This code acts before the modification and the modification is already in a `runInAction` (at line 635).
So, this code not need `runInAction`.